src/app/grid-cellEditing/grid-cellEditing.component.html

```diff
@@ -1,41 +1,130 @@
+<div class="grid__wrapper">
+    <div class="sample__header">
+        <!-- <app-grid-with-transactions> -->
+            <igx-grid igxPreventDocumentScroll #grid12 [data]="data1" [autoGenerate]="false" displayDensity="cosy"
```

Comment:
Just keep in mind that we should remove the sample here as we have almost the same defined into the row editing section.

Reply:
Once @damyanpetev and @rkaraivanov review and approve the changes I will remove it along with the "do not merge" label.